docs/12-self-hosting/03-host-creation/03-QEMU/05-macos.mdx

```diff
@@ -0,0 +1,280 @@
+import Vnc from '/assets/images/vnc-connection.png';
+import Ssh from '/assets/images/ssh-connection.png';
+import InstallLocation from '/assets/images/macos-choose-install-location.png';
+import FormatDisk from '/assets/images/macos-format-disk.png';
+import QuitDisk from '/assets/images/macos-quit-disk-utility.png';
+import Reinstall from '/assets/images/macos-reinstall.png';
+import CreateAccount from '/assets/images/macos-create-account.png';
+import BootMedia from '/assets/images/macos-select-boot-media.png';
+import BootMedia2 from '/assets/images/macos-select-boot-media2.png';
+import DiskUtility from '/assets/images/macos-select-disk-utility.png';
+import InstallMedia from '/assets/images/macos-select-install-media.png';
+import MacosSsh from '/assets/images/macos-ssh.png';
+import RemoteLogin from '/assets/images/macos-remote-login.png';
+import Hackintosh from '/assets/images/macos-hackintosh.png';
+
+# MacOS
+
+For MacOS guests, we will utilize the open-source project
+[OSX-KVM](https://github.com/kholia/OSX-KVM) which will help you create a **Virtual Hackintosh**.
+Such a system can be used for a variety of purposes (e.g. software builds, testing, reversing work)
+However, such a system lacks graphical acceleration, a reliable sound sub-system, USB 3
+functionality and other similar things. To enable these things, take a look their
+[notes](https://github.com/kholia/OSX-KVM/blob/master/notes.md). Older AMD CPU(s) are known to be
+problematic but modern AMD Ryzen processors work just fine.
```

Comment:
We should probably mention something about legal considerations as running macOS on non-Apple hardware is against Apple's End User License Agreement (EULA).

Reply:
added in commit https://github.com/game-ci/documentation/pull/419/commits/a2d56f70d9183671d0063b0127a099b2c8bc1eb5